src/extension/importer/core/src/test/java/org/geoserver/importer/ImporterDataTest.java

```diff
@@ -650,6 +651,34 @@ public void testImportGeoTIFFFromDataDir() throws Exception {
                 ((CoverageStoreInfo) task.getLayer().getResource().getStore()).getURL());
 
         runChecks("EmissiveCampania");
+        CoverageStoreInfo store = cat.getCoverageStoreByName("EmissiveCampania");
+
+        // Replacing the layer
+        File tempDir = FileUtils.getTempDirectory();
+        dir = unpack("geotiff/EmissiveCampania.tif.bz2", tempDir);
```

Comment:
If it's the same source file, how do you know the replace actually worked?

Reply:
I'm actually updating the same file but from a different path and the storeURL get properly changed.